src/components/ItemTooltip/index.js

```diff
@@ -61,6 +62,13 @@ export default function ItemTooltip({ item, small, dismiss, globalItemCount }) {
             objectives={item.$objectives}
           />
         )}
+
+        {item.$statTrack && (
+          <StatTrack
+            className={styles.objectives}
+            statTrack={item.$statTrack}
```

Comment:
So here, just make sure you give this a meaningful `className` to style the text properly.

I think as long as it matches the style and colour of the description that should be good. In fact, maybe even `className={styles.description}` would be good enough 😄